ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -86,12 +86,24 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   const visual = merge({}, DEFAULT_VISUAL, props.spec.visual);
 
   // convert Perses dashboard format to be ECharts compatible
-  const yAxis = {
+  const yAxis: YAXisComponentOption = {
     show: y_axis?.show ?? DEFAULT_Y_AXIS.show,
-    min: y_axis?.min, // leaves min and max undefined by default to let ECharts calcualate
+    min: y_axis?.min,
     max: y_axis?.max,
   };
 
```

Comment:
This really would be a nice case to add a few stories in storybook, so we could get visual diffs for this behavior, but the time series data mocking util I wrote is still super-simple and locked into the e2e package. I'm going to take a little time today to see if I can pull those out into something we'd be able to use across both storybook and playwright.

Reply:
We also don’t have any panel stories yet right, mostly just component-level? I wasn’t sure if there was some other blocker to getting a full panel working, I can give it a shot though!

Reply:
I added a panel story in storybook in one of my feature branches, not merged yet though

Reply:
Added a story with examples of customizing a panel's y_axis spec and used a query that showed how the new origin logic works.